audiophiler/templates/main.html

```diff
@@ -80,6 +80,36 @@ <h6>Owner: {{ file.author }} uploaded {{ get_date_modified(s3_bucket, file.file_
             {% if not loop.index0 is divisibleby 2 %}
                 </div>
             {% endif %}
+            {% if loop.index0 is divisibleby 2 and loop.last %}
+                </div>
+            {% endif %}
         {% endfor %}
+        <div class="col" align="center">
+            <nav aria-label="page selector">
+                <ul class="pagination pagination-lg justify-content-center">
+                    {% if page == 1 %}
+                    <li class="page-item disabled"> 
+                        <a class="page-link" href="{{ current }}{{ page - 1 }}" tabindex="-1">Previous</a>
```

Comment:
would `request.url` work instead of current? doc: https://flask.palletsprojects.com/en/2.0.x/templating/